disconnected/install-disconnected.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="install-disconnected"]
+= Installing {product-title} in a disconnected environment
+:context: install-disconnected
+
+toc::[]
+
+Before you can install {product-title} on a disconnected {platform} cluster, you must enable Operator Lifecycle Manager (OLM) in disconnected environments, by completing the following steps:
+
+* Disable the default remote OperatorHub sources for OLM.
```

Comment:
I see that attributes exists for OLM:

<img width="714" height="107" alt="Screenshot From 2025-07-24 16-53-50" src="https://github.com/user-attachments/assets/a0bd35cb-1afb-42ac-8a2d-e03fa2cf38be" />